src/pages/search.tsx

```diff
@@ -1,19 +1,30 @@
 import React, { ReactElement } from 'react'
 import PageSearch from '../components/templates/Search'
-import { PageProps } from 'gatsby'
+import { navigate, PageProps } from 'gatsby'
 import Layout from '../components/Layout'
 import queryString from 'query-string'
+import ethereumAddress from 'ethereum-address'
+import { accountTruncate } from '../utils/wallet'
 
 export default function PageGatsbySearch(props: PageProps): ReactElement {
   const parsed = queryString.parse(props.location.search)
-  const { text, tags, categories } = parsed
+  const { text, owner, tags, categories } = parsed
+
+  // Switch to owner search when ETH address in `text` is detected,
```

Comment:
This can only be a temporary hack and ideally we do. to ship this. Proper way is making Aquarius search through everything when `text` query is used https://github.com/oceanprotocol/aquarius/issues/304